src/machine/behavior/is/mod.rs

```diff
@@ -19,15 +19,263 @@ pub fn get(_opcode: u8) -> Semantic {
 }
 
 type Semantic = fn(&mut State, u8, u8, u8);
-/*
-pub const SEMANTICS: [Semantic; 256] = [
-                        // symbol   hex  dec
-
-    interrupt::trap,    // TRAP     #00    0
-    float::fcmp,        // FCMP     #01    1
-
-    /* TODO */          // PUSHGOI  #BF  191
 
-    other::swym         // SWYM     #FF  255
+pub const SEMANTICS: [Semantic; 256] = [
+    interrupt::trap,    
```

Comment:
Yes, as written in the pull request's description, if you feel that this is worth two hours of you your time, then you can do that. If you see a ways to do it faster, please tell me ;)